#! /bin/sh /usr/share/dpatch/dpatch-run
## pod-errors.dpatch by Sebastian Harl <sh@tokkee.org>
##
## DP: collectd(1), collectd{-exec,.conf}(5): Fixed some minor POD errors.

@DPATCH@

diff a/src/collectd-exec.pod b/src/collectd-exec.pod
--- a/src/collectd-exec.pod
+++ b/src/collectd-exec.pod
@@ -171,11 +171,13 @@ table. All the options are optional, but B<plugin_instance> without B<plugin>
 or B<type_instance> without B<type> doesn't make much sense and should be
 avoided.
 
-Please note that this is the same format as used in the B<unixsock plugin>, see
-L<collectd-unixsock(5)>.
+=back
 
 =back
 
+Please note that this is the same format as used in the B<unixsock plugin>, see
+L<collectd-unixsock(5)>.
+
 When collectd exits it sends a B<SIGTERM> to all still running
 child-processes upon which they have to quit.
 
diff a/src/collectd.conf.pod b/src/collectd.conf.pod
--- a/src/collectd.conf.pod
+++ b/src/collectd.conf.pod
@@ -1323,6 +1323,8 @@ This option, although numeric, needs to be a string, i.E<nbsp>e. you B<must>
 use quotes around it! If no such statement is given only global information
 will be collected.
 
+=back
+
 =head2 Plugin C<tcpconns>
 
 The C<tcpconns plugin> counts the number of currently established TCP
@@ -1333,8 +1335,6 @@ fine-tune the ports you are interested in:
 
 =over 4
 
-=back
-
 =item B<ListeningPorts> I<true>|I<false>
 
 If this option is set to I<true>, statistics for all local ports for which a
diff a/src/collectd.pod b/src/collectd.pod
--- a/src/collectd.pod
+++ b/src/collectd.pod
@@ -116,6 +116,8 @@ internal caches. E.E<nbsp>g. the C<rrdtool plugin> will write all pending data
 to the RRD files. This is the same as using the C<FLUSH -1> command of the
 C<unixsock plugin>.
 
+=back
+
 =head1 SEE ALSO
 
 L<collectd.conf(5)>,
